### PR TITLE
web optimize for 1.4.0

### PR DESCRIPTION
**Pull request by mayinrain**
### What is the purpose of the change

fix known web bugs of dev-1.4.0

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Review by casionone — APPROVED** [review]
LGTM

File: linkis-web/src/apps/linkis/module/datasourceAccess/index.vue [hunk 1]
```diff
@@ -242,6 +242,7 @@ export default {
                 duration: 3,
                 content: "添加成功"
```
Comment (det101) [inline linkis-web/src/apps/linkis/module/datasourceAccess/index.vue:243]: It is recommended to put it in i18n

Comment (casionone) [reply linkis-web/src/apps/linkis/module/datasourceAccess/index.vue:243]: @mayinrain

**Comment by jacktao007** [comment]
Please resolve the conflict

File: linkis-web/src/apps/linkis/module/datasourceAccess/index.vue (hunk 1 above)
Comment (mayinrain) [reply linkis-web/src/apps/linkis/module/datasourceAccess/index.vue:243]: Thank you for your suggestion! Currently, the access to this page is still blocked, and internationalization optimization will be performed upon the release of future features.

**Pull request merged**